### ISSUE TITLE
No way to hide switches?

### ISSUE THREAD

**Issue by letsstartbeinganon** [DigiLive/mushroom-strategy#149]
Hi, I've used Mushroom Strategy to generate a dashboard that looks great but there's a lot of information present that I don't need. I've managed to hide a lot of entities that I don't need to see but there doesn't seem to be a way to hide things that are shown as switches? How do I do so?

![Screenshot_5](https://github.com/user-attachments/assets/8a42d0a8-95df-4b63-b71b-37fa37a1c041)

**Comment by DigiLive** [comment]
as per Wiki (https://github.com/AalianKhan/mushroom-strategy/wiki/Strategy-Options#card-options):

```yaml
strategy:
  type: custom:mushroom-strategy
  options:
    card_options:
      switch.nest_protect_gang_heads_up:
        hidden: true
```

**Comment by erocm123** [comment]
I can confirm that the method mentioned by @DigiLive works (at least for the lights I have tested). 

One thing I have noticed is if I hide an entity in this way, it still shows up as part of the view counter and using the view counter icon button to turn all lights off or on will still apply to those hidden entities. 

![image](https://github.com/user-attachments/assets/fc96937f-ea30-4781-a6ea-4a89ef96f332)


![image](https://github.com/user-attachments/assets/04c6954f-69af-40e6-84ce-c67a4025eaa0)

Shouldn't they be excluded if they are hidden? Sorry if this is too far off topic and I can make a new issue.
